packages/action-password-update/src/components/PasswordUpdateDialog/PasswordUpdateDialog.scss

```diff
@@ -1,54 +1,206 @@
-.PasswordUpdateDialog {
+/*****
+* import sass dependencies
+*/
 
-  .oldPassword {
-    margin-bottom: 1.5em;
-  }
+// TODO reduce redundancy
+// TODO get rid of cumbersome imports
+// TODO make it possible to use _variables.scss from customer
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_opacity.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_background-variant.scss";
+@import "../../../../../node_modules/font-awesome/scss/_variables.scss";
 
-  .ValidationRules {
-    margin-bottom: 1.5em;
-  }
+$fa-font-path: "../../../../../node_modules/font-awesome/fonts/";
+@import "../../../../../node_modules/font-awesome/scss/font-awesome.scss";
+
+
+/*****
+* tocco-client boilerplate
+* for styling convenience while developing, only used inside monorepo tocco-client
+*/
+
+// TODO replace #root with #tocco-client-development
```

Comment:
why do you want to change this id?

And should this rule really be placed in this file? It's only for the development environment.